tests/common/Microsoft.DotNet.UpgradeAssistant.Abstractions.Tests/TargetFrameworkMonikerTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+using static Microsoft.DotNet.UpgradeAssistant.TargetFrameworkMonikerParser;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Abstractions.Tests
+{
+    public class TargetFrameworkMonikerTests
+    {
+        public static IEnumerable<object[]> EqualityData()
+        {
+            yield return new object[] { TargetFrameworkMoniker.Net45, TargetFrameworkMoniker.Net45 };
+            yield return new object[] { TargetFrameworkMoniker.Net45, TargetFrameworkMoniker.Net45 with { Framework = "NET" } };
```

Comment:
why use "NET" here instead of the constant? were you testing case sensitivity?

Reply:
Case sensitivity and framework name normalization. I've add a comment to what the value is.